spec/lib/hubspot/ticket_properties_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe Hubspot::TicketProperties do
+  describe '.create' do
+    context 'with no valid parameters' do
```

Comment:
please start with a happy path
also
```suggestion
    context 'with invalid parameters' do
```